README.rst

```diff
@@ -114,22 +115,22 @@ Now, install and run ``raincoat`` in your project:
 .. code-block:: console
 
     $ pip install raincoat
-	$ raincoat
+    $ raincoat
 
 
 It will:
 
-- Grep the code for all ``# Raincoat:`` comments and for each comment:
-- Look at the currently installed version of the lib (say, umbrella 16.0.3) (or, if not
-  found, the latest version)
-- Compare with the version in the Raincoat comment (here, 14.5.7)
-- If they are different, download and pip install the specified version in a temp dir
-  (using cached wheel as pip does by default, this should be quite fast in most cases)
-- Locate the code using the provided path for both the downloaded and the currently
-  installed versions
-- Diff it
-- Tell you if there's a difference (and mention the location of the original Raincoat
-  comment)
+#. Grep the code for all ``# Raincoat:`` comments and for each comment.
+#. Look at the currently installed version of the lib (say, umbrella 16.0.3) (or, if not
+   found, the latest version)
+#. Compare with the version in the Raincoat comment (here, 14.5.7)
+#. If they are different, download and pip install the specified version in a temp dir
+   (using cached wheel as pip does by default, this should be quite fast in most cases)
+#. Locate the code using the provided path for both the downloaded and the currently
+   installed versions
+#. Diff it
+#. Tell you if there's a difference (and mention the location of the original Raincoat
+   comment)
```

Comment:
I would keep the colon and then indent the following items.
```
- Grep the code for all ``# Raincoat:`` comments and for each comment:
  #. Look at the currently installed version of the lib (say, umbrella 16.0.3) (or, if not
     found, the latest version)
  #. Compare with the version in the Raincoat comment (here, 14.5.7)
  #. If they are different, download and pip install the specified version in a temp dir
     (using cached wheel as pip does by default, this should be quite fast in most cases)
  #. Locate the code using the provided path for both the downloaded and the currently
     installed versions
  #. Diff it
  #. Tell you if there's a difference (and mention the location of the original Raincoat
     comment) ```

Reply:
Agree. 👍